aioftp/client.py

```diff
@@ -547,13 +697,38 @@ def list(self, path="", *, recursive=False):
 
             >>> stats = await client.list()
         """
-        class AsyncClientLister(AsyncListerMixin):
+        class AsyncListerClient(AsyncListerMixin):
```

Comment:
Why name is changed? I'm not native English, but this looks wrong. My naming was like "asynchronous ftp client lister". Maybe `AsyncLister` is enough here.